src/app/core/config/config-fix.ts

```diff
@@ -940,6 +931,8 @@ export const defaultJsonConfig = {
   },
 
   "entity:Child": {
+    "label": "Participant",
```

Comment:
We either have to remove this or use `$localize` here otherwise the translation in the demo setup doesn't work.